pegasus/sites.v3/code.org/public/minecraft.haml

```diff
@@ -51,6 +51,27 @@ style_min: true
   =I18n.t(:minecraft_no_internet)
   %a{href: CDO.studio_url("/download/mc")}= I18n.t(:minecraft_download_adventurer)
 
+.color-block.mc-store
+  %h2.white-text= I18n.t(:minecraft_store_camps)
+  = I18n.t(:minecraft_signup_store_camps).gsub('%{signup_store_camp}', 'https://www.microsoft.com/youthsparkprograms')
+
+.color-block.mc-host-event
+  %h2.white-text= I18n.t(:minecraft_host_event)
+  %a{href: '/hourofcode/mc'}= I18n.t(:minecraft_teach_resources)
+  %br
+  %a{href: 'https://docs.com/hourofcode2016'}= I18n.t(:minecraft_leader_guide)
```

Comment:
@tanyaparker Do these resources need to be updated for 2017?

Reply:
Yes, that looks old. Have we shared back these boxes with Microsoft? @ryansloan I think we need to update these resources?

Reply:
All the other links seem to be ok to me.